airbyte-integrations/connectors/source-postgres/integration_tests/abnormal_state_copy.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
Kind of a nit: since this is going to be overwritten during setup, does it make sense to call it abnormal_state_sample.json or abnormal_state_template.json (and then interpolate over the a placeholder value wherever one needs to be replaced)? It might avoid future confusion if we indicate that it won't be used in its raw form.

(Same comment for the configured catalog files.)

Reply:
good point. fixed!